rfcs/rfc-business-ledger-data-structure.md

```diff
@@ -0,0 +1,83 @@
+- Start Date: 2021-04-22
+- Target Major Version: [information on a ledger](https://github.com/bcgov/entity/issues/6969)
```

Comment:
This is the epic but not the version. What is expected here?

Reply:
Usually what release we're planning this for, and it should align with this epic, hence I chose that.